code/datums/mutations/hulk.dm

```diff
@@ -50,7 +50,7 @@
 	name = "Hulk"
 	desc = "A seemingly dormant genome, but reacts violently to aggitation."
 	difficulty = 16
-	instability = 50
```

Comment:
I think 50 is fine imo

Reply:
Stacking hulk with thick skin very easily is not a good thing

Reply:
```
owner.physiology.brute_mod *= 0.8
owner.physiology.burn_mod *= 0.9
```
oh no a 10/20% reduction?

Reply:
ugh fine 🙄